src/core/server/saved_objects/service/lib/repository.test.js

```diff
@@ -27,7 +27,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
+import { omit } from 'lodash';
```

Comment:
saw comment from Rocky we'll remove lodash?

Reply:
Yes, we should do that.